src/test/java/org/mule/tools/apikit/MainAppScaffolderWithExistingConfigApiSyncTest.java

```diff
@@ -65,72 +76,76 @@ public void reScaffold() throws Exception {
     ScaffoldingConfiguration configuration =
         new ScaffoldingConfiguration.Builder().withApi(parseResult.get()).build();
     ScaffoldingResult result = mainAppScaffolder.run(configuration);
-    assertTrue(result.isSuccess());
-    assertEquals(1, result.getGeneratedConfigs().size());
 
-    String expected = APIKitTools.readContents(getResourceAsStream("rescaffolding-apisync-version/v1/api.xml"));
-    String generated = APIKitTools.readContents(result.getGeneratedConfigs().get(0).getContent());
-
-    XMLUnit.setIgnoreWhitespace(true);
-    Diff diff = XMLUnit.compareXML(expected, generated);
-    assertTrue(diff.identical());
+    verifySuccessfulScaffolding(result, "rescaffolding-apisync-version/v1/api.xml");
```

Comment:
It seems that `rescaffolding-apisync-version/v1/api.xml` should be a constant, the path is spread all over the file